frontend/src/components/Table/Tab.tsx

```diff
@@ -0,0 +1,223 @@
+//     const [dataSource, setDataSource] = useState<DataSourceItem[]>([]);
+//
+//     useEffect(() => {
+//         if (!resourceLabel) return;
+//         console.log(resourceLabel);
+//
+//         const fetchData = async () => {
+//             const response: ApiResponse = await fetchResources(resourceLabel);
+//
+//             const dynamicColumns: ColumnType[] = response.columns.map((column) => ({
+//                 title: column,
+//                 dataIndex: column,
+//                 key: column,
+//                 width: 150,
+//                 render: (text: React.ReactNode, record: DataSourceItem): React.ReactNode => {
+//                     if (column.toLowerCase().includes('age')) {
+//                         return formatAge(record[column] as string);
+//                     }
+//                     return text;
+//                 },
+//             }));
+//
+//             dynamicColumns.push({
+//                 dataIndex: "",
+//                 title: 'Actions',
+//                 key: 'actions',
+//                 render: (_, record: DataSourceItem) => (
+//                     <div>
+//                         <Button
+//                             type="link"
+//                             icon={<EditOutlined />}
+//                             onClick={() => handleEdit(record)}
+//                         />
+//                         <Button
+//                             type="link"
+//                             icon={<DeleteOutlined />}
+//                             onClick={() => handleDelete(record)}
+//                             danger
+//                         />
+//                     </div>
+//                 ),
+//                 width: 100
+//             });
+//
+//             const dynamicDataSource: DataSourceItem[] = response.resource_list.map((resource, index) => ({
+//                 key: index,
+//                 ...resource,
+//             }));
+//
+//             setColumns(dynamicColumns);
+//             setDataSource(dynamicDataSource);
+//         };
+//
+//         fetchData();
+//     }, [resourceLabel]);
+//
+//     const handleAdd = () => {
+//         console.log("POST");
+//     };
+//
+//     const handleEdit = (record: DataSourceItem) => {
+//         console.log("PUT", record);
+//     };
+//
+//     const handleDelete = (record: DataSourceItem) => {
+//         console.log("DELETE", record);
+//     };
+//
+//     return (
+//         <div>
+//             <Button
+//                 type="primary"
+//                 icon={<PlusOutlined />}
+//                 onClick={handleAdd}
+//                 style={{ marginBottom: 16 }}
+//             >
+//                 Add
+//             </Button>
+//             <Table
+//                 columns={columns}
+//                 dataSource={dataSource}
+//                 scroll={{ x: 'max-content', y: 55 * 5 }}
+//             />
+//         </div>
+//     );
+// };
+//
+// export default Tab;
```

Comment:
Not sure why this is commented. Both look nearly identical.

Reply:
Needed for comparing two versions